spark/build.gradle.kts

```diff
@@ -0,0 +1,113 @@
+plugins {
+  `maven-publish`
+  id("java")
+  id("scala")
+  id("idea")
+  id("com.diffplug.spotless") version "6.11.0"
+  signing
+}
+
+publishing {
+  publications {
+    create<MavenPublication>("maven-publish") {
+      from(components["java"])
+
+      pom {
+        name.set("Substrait Java")
+        description.set(
+          "Create a well-defined, cross-language specification for data compute operations"
+        )
+        url.set("https://github.com/substrait-io/substrait-java")
+        licenses {
+          license {
+            name.set("The Apache License, Version 2.0")
+            url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
+          }
+        }
+        developers {
+          developer {
+            // TBD Get the list of
```

Comment:
Hmm, we should have this somewhere.

Reply:
I just copied the pom section from the other build files in the project.  We could just remove the developer section entirely.